task-94/task-94.js

```diff
@@ -96,10 +97,13 @@ function moveBack(id) {
     clearTimeout(timeoutId);
     let site = document.getElementById(id);
     let i = (listOfActive.findIndex(x => x == site.src)); 
+
     if (i <= 0) {
-        site.src = listOfActive[0]
+        site.src = listOfActive[0];
+        location.hash = `#window0`
      } else {
         site.src = listOfActive[(i-1)];  
+        location.hash = `#window${i-1}`
```

Comment:
Unfortunately, this approach won't work properly when changing url hash part manually.

Reply:
An example of simple Router implementation
https://gist.github.com/ArturJS/9cc26c5821279f4b0348adf12ba7113f

Reply:
@Lu-el The idea of my proposal was to replace `<iframe src="https://www.b17.ru/blog/101157/" name="mainWindow" frameborder="0" id="mainWindow" onload="moveTime()"></iframe>` with  `<div id="root"></div>` in task94.html, plus inside 
```
render(){
...
return `<div> Page ${i}</div>`;
}
```

replace  `<div> Page ${i}</div>`; with iframe elements, so `pages` parameter of `new Router` will contain multiple objects with different iframes that in turn will be mounted inside <div id="root"> element according to the logic inside class Router.